zboxapi/sdk.go

```diff
@@ -209,3 +235,36 @@ func (c *Client) RefreshJwtToken(ctx context.Context, phoneNumber string, token
 
 	return result.Token, nil
 }
+
+func (c *Client) GetFreeStorage(ctx context.Context, phoneNumber, token string) (string, error) {
```

Comment:
maybe we can add in the name, that this particular func can be used only with jwtToken, something like `GetFreeStorageWithJWT`

Reply:
it seems not be better. Because `phonenumber` is required too. so would you like to rename it with `GetFreeStorageWith`?